### PR TITLE
Finish stylized text prototype

### PR DESCRIPTION
**Pull request by WeijiaMa**
@WeijiaMa Please include a description of the PR here!

### CODE REVIEW

File: src/main/java/codeu/controller/ChatServlet.java [hunk 1]
```diff
@@ -140,8 +140,8 @@ public void doPost(HttpServletRequest request, HttpServletResponse response)
 
     String messageContent = request.getParameter("message");
 
-    // this removes any HTML from the message content
-    String cleanedMessageContent = Jsoup.clean(messageContent, Whitelist.none());
+    // this removes any illegal HTML from the message content
+    String cleanedMessageContent = Jsoup.clean(messageContent, Whitelist.basic());
```
Comment (dgalbraith33) [inline src/main/java/codeu/controller/ChatServlet.java]: nit: Can you elaborate in this comment about what HTML is whitelisted?

File: src/test/java/codeu/controller/ChatServletTest.java [hunk 2]
```diff
@@ -207,8 +207,9 @@ public void testDoPost_CleansHtmlContent() throws IOException, ServletException
     ArgumentCaptor<Message> messageArgumentCaptor = ArgumentCaptor.forClass(Message.class);
     Mockito.verify(mockMessageStore).addMessage(messageArgumentCaptor.capture());
     Assert.assertEquals(
-        "Contains html and  content.", messageArgumentCaptor.getValue().getContent());
+        "Contains \n<b>html</b> and  content.", messageArgumentCaptor.getValue().getContent());
```
Comment (dgalbraith33) [inline src/test/java/codeu/controller/ChatServletTest.java]: Why the '\n' newline in this assert?

**Review by dgalbraith33 — CHANGES_REQUESTED** [review]

File: src/test/java/codeu/controller/ChatServletTest.java (hunk 2 above)
Comment (tiffanycwong) [reply src/test/java/codeu/controller/ChatServletTest.java]: +1

File: src/test/java/codeu/controller/ChatServletTest.java [hunk 3]
```diff
@@ -207,8 +207,9 @@ public void testDoPost_CleansHtmlContent() throws IOException, ServletException
...
 
     Mockito.verify(mockResponse).sendRedirect("/chat/test_conversation");
   }
+
```
Comment (tiffanycwong) [inline src/test/java/codeu/controller/ChatServletTest.java]: nit: remove this extra line

**Review by tiffanycwong — CHANGES_REQUESTED** [review]

**Review by tiffanycwong — APPROVED** [review]

File: src/test/java/codeu/controller/ChatServletTest.java (hunk 3 above)
Comment (tiffanycwong) [reply src/test/java/codeu/controller/ChatServletTest.java]: oops. connecting on this because i messed up with github :P

**Review by tiffanycwong — CHANGES_REQUESTED** [review]

**Comment by WeijiaMa** [comment]
I have updated with the changes @dgalbraith33 @tiffanycwong

**Review by tiffanycwong — APPROVED** [review]

**Comment by tiffanycwong** [comment]
Please add description up top!

**Pull request merged**